charts/psmdb-db/values.yaml

```diff
@@ -0,0 +1,127 @@
+# Default values for pxc-cluster.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Platform type: kubernetes, openshift
+# platform: kubernetes
+
+# Cluster DNS Suffix
+# DNSsuffix: .svc.cluster.local
+
+pause: false
+allowUnsafeConfigurations: false
+# updateStrategy: SmartUpdate
```

Comment:
more fields are needed for smartupdate so maybe add them commented if it is going to be disabled here: https://github.com/percona/percona-server-mongodb-operator/blob/master/deploy/cr.yaml#L14-L18

Reply:
done